template/src/[namespace]-[name].js

```diff
@@ -30,11 +30,13 @@ class [Namespace][Name] extends LitElement {
   //   super();
   // }
 
-  // function to define props used within the scope of this component
+  // This function is to define props used within the scope of this component
+  // Be sure to review  https://lit-element.polymer-project.org/guide/properties#reflected-attributes
+  // to understand how to use reflected attributes with your property settings.
   static get properties() {
     return {
       // ...super.properties,
-      cssClass:   { type: String }
+      cssClass:   { type: String } // this property is DEMO ONLY! Please delete.
```

Comment:
How about just commenting this line out like line 38?

Reply:
It can't be commented out because the template needs the prop to render the UI. 

I put the comment next to it like this specifically to draw attention to this.